src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Navbar.css';
+
+const Nav = () => {
+  const navStyle = {
+    color: 'white',
+    textDecoration: 'none',
+  };
+
+  return (
+    <nav className="nav">
+      <h1>Math Magician</h1>
+      <ul className="nav-ul">
+        <Link style={navStyle} to="/home">
+          <li>Home</li>
+        </Link>
```

Comment:
- [ ] As I discussed above to use the root route for the Home component, You also need to fix your navbar. Please change the link for the Home button from `to="/home"` to this `to="/"`.